homeassistant/components/abode/__init__.py

```diff
@@ -294,6 +303,11 @@ def __init__(self, data, device):
             self._data.abode.events.remove_all_device_callbacks, self._device.device_id
         )
 
+    @property
```

Comment:
Looks like you are adding the same code into similar entities.  Can you save a few lines by using a base class for the shared code?